server/controllers/stripe.js

```diff
@@ -0,0 +1,144 @@
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+require('dotenv').config();
+const stripe = require("stripe")(process.env.STRIPE_SK);
+
+const createCustomer = async(profile, email) => {
+  const customer = await stripe.customers.create({
+    name: profile.name,
+    address: profile.address,
+    email: email,
+  });
+  if(!customer){
+    throw new Error("Unable to create a customer, please try again!")
+  } else {
+    profile.customerId = customer.id
+
+    let savedProfile = await profile.save()
```

Comment:
```suggestion
    const savedProfile = await profile.save()
```
I don't see us reassigning this variable